### ISSUE TITLE
"Better Pawn Control" RT 1.3.1 and Better Pawn Control schedules

### ISSUE THREAD

**Issue by Numenar** [cseelhoff/RimThreaded#448]
**Describe the bug**
RT 1.3.1 and Better Pawn Control schedules incompatibility (I don't use other features)

Fresh start throws red lines, stops functionality of Emergency Button schedules switching.
On an old save with ~50 colonists it still switched some but not all schedules.

**To Reproduce (VERY IMPORTANT)**
1. Make 2 schedules
2. Setup switching schedules in Better Pawn Control config
3. Click on Emergency Button Icon repeatedly, it will stop responding and starts throwing error
4. After that you can't open schedules tab (same error) you have to unpause game first.

**Error Log**
* Link to contents of Player.log  https://file.io/8fxPnL8ozrmU
https://gist.github.com/HugsLibRecordKeeper/51e19dfcf09a0f2e231342e6e5186331

**Mod List**
[Mod list length: 7]
Harmony [https://steamcommunity.com/sharedfiles/filedetails/?id=2009463077]
BetterLoading [https://steamcommunity.com/sharedfiles/filedetails/?id=1999454301]
Core [https:// no link (local mod)]
Royalty [Official DLC] [https:// no link (local mod)]
HugsLib [https://steamcommunity.com/sharedfiles/filedetails/?id=818773962]
Better Pawn Control [https://steamcommunity.com/sharedfiles/filedetails/?id=1541460369]
RimThreaded [1.2] [https://steamcommunity.com/sharedfiles/filedetails/?id=2222907981]

**Comment by Eldulac** [comment]
**[Tested on RT 1.3.2]**

I tested this issue on a couple of fresh games, with both tribal and crashlanded starts.
I was not able to reproduce the described problem, despite several attempts. I spawned dozens of pawns and set up three different schedules for them. Pressing the emergency button repeatedly, and switching schedules manually, both worked as designed.
On one isolated case though, the "Auto" schedule seemed to revert to one of the other schedules, in terms of time allocation.
This may have been due to a misclick on my part (copy and paste), or an automated feature of the mod, but this only happened once during the test.

Here is a test save for future reference:
[BetterPawnControl Schedules.zip](https://github.com/cseelhoff/RimThreaded/files/6170413/BetterPawnControl.Schedules.zip)

**Conclusion**
"BetterPawnControl" schedules seem to be working correctly, and none of the described issues were found.
